Added known limitations documentation (#22)
CitrineInformatics/taurus-documentation: docs/specification/objects.md

---
## Ingredient Run

> A particular instance of an ingredient spec. Note that the `name` and `labels` for an Ingredient Spec are shared with all associated Ingredient Runs.

Reviewer: ```suggestion
A particular instance of an ingredient spec.
Note that the `name` and `labels` for an Ingredient Run are inherited from its spec.
```

> *Due to complexities of implementation, care must be taken in constructing Ingredient Runs.*  See [Known Limitations](../../known_limitations).  

Reviewer: `care must be taken` seems unnecessarily opaque.  You could reference the change to this spec when name/labels were removed and mention that some implementations may still have name/labels on `IngredientRun`.
